styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -170,6 +177,14 @@ void tick() {
     eligibleInstances.forEach(limitAndDequeue);
   }
 
+  private void updateMetrics(Map<String, Resource> resources,
+                             Map<String, Long> currentResourceUsage) {
+    resources.values().forEach(r -> stats.resourceConfigured(r.id(), r.concurrency()));
+    currentResourceUsage.forEach(stats::resourceUsed);
+    Sets.difference(resources.keySet(), currentResourceUsage.keySet())
+        .forEach(r -> stats.resourceUsed(r, 0));
```

Comment:
That should have been included in `resources`. https://github.com/spotify/styx/pull/194/files#diff-a473be0c753046d2e7f56a00c3ed5b62L112